Makefile

```diff
@@ -32,17 +32,17 @@ clean:
 
 install:
 	mkdir -m 0755 -p $(VAR) $(SBIN) $(MAN) $(ETC) $(BASHDIR) $(SHAREDIR) $(LIBDIR)
-	sed -e "s/\[INSERT_VERSION_HERE\]/$(RELEASE_VERSION)/" dkms > dkms.versioned
-	mv -f dkms.versioned dkms
 	install -p -m 0755 dkms_common.postinst $(LIBDIR)/common.postinst
 	install -p -m 0755 dkms $(SBIN)
 	install -p -m 0755 dkms_autoinstaller $(LIBDIR)
 	install -p -m 0644 dkms_framework.conf $(ETC)/framework.conf
 	install -p -m 0644 dkms_dbversion $(VAR)
 	install -p -m 0644 dkms.bash-completion $(BASHDIR)/dkms
-	# install compressed manpage with proper timestamp and permissions
-	gzip -c -9 dkms.8 > $(MAN)/dkms.8.gz
-	chmod 0644 $(MAN)/dkms.8.gz
+	install -p -m 0644 dkms.8 $(MAN)
+	# add versioning
+	sed -i -e 's/RELEASE_VERSION/$(RELEASE_VERSION)/' -e 's/RELEASE_STRING/$(RELEASE_STRING)/' -e 's/RELEASE_DATE/$(RELEASE_DATE)/' $(SBIN)/dkms $(MAN)/dkms.8
```

Comment:
Would you mind to use some marking characters like ## around the match string?  This helps to prevent inadvertently causing weird behavior down the road when someone forgets this sed statement is part of `make install`.